packages/endace/_dev/build/build.yml

```
@@ -0,0 +1,4 @@
+dependencies:
+  ecs:
+    reference: "git@v8.11.0"
```

```suggestion
    reference: "git@v8.17.0"
```

ECS 8.17.0 is available now, and should be used